views/books.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Book List</title>
+  <link rel="stylesheet" type="text/css" href="stylesheets/books.css" />
+</head>
+<body>
+  <h1>Reading List</h1>
+  <h3>Currently Reading</h3>
+  <p id="currently-reading">
+    <!-- currently reading will be rendered here -->
```

Comment:
CoPilot initially added this comment but I think I like having all of them so you can tell what is going on.